Spanish translation

DeimosHall opened the pull request.

DeimosHall committed:
Add Spanish translation for contact page

DeimosHall committed:
Fix title name translation

DeimosHall committed:
Add Spanish translation for credits page

DeimosHall committed:
Add Spanish translation for Making ASICs page

DeimosHall committed:
Add Spanish translation for Tiny Tapeout 01 page
- Move images from runs/tt01/images to static/images

DeimosHall committed:
Add Spanish translation for submit page

sabas1080 reviewed (commented):
Este pullrequest tambien incluye los del pullresquest de creditos, solo tener cuidado en los siguientes commits no incluir los cambios pasados, solo los nuevos

DeimosHall committed:
Add Spanish translation for success page

DeimosHall committed:
Merge branch 'ElectronicCats:main' into spanish_translation

DeimosHall committed:
Add Spanish translation for Working with HDLs page

DeimosHall committed:
Add Spanish translation for fpga vs asic page from hdl page

DeimosHall committed:
Merge branch 'spanish_translation' of https://github.com/DeimosHall/tinytapeout_www into spanish_translation

DeimosHall committed:
Add Spanish translation for important page from hdl page

DeimosHall committed:
Add Spanish translation for hdl resources page

DeimosHall committed:
Add Spanish translation for hdl templates page

DeimosHall committed:
Add Spanish translation for testing your design page

DeimosHall committed:
Add Spanish translation for digital design guide page

DeimosHall committed:
Add Spanish translation for wokwi page

DeimosHall committed:
Add Spanish translation for the faq page
- Move images from faq/images to static/images/faq
- Change images path on the english version

DeimosHall committed:
Add Spanish translation for customisable design 7-Seg page

DeimosHall committed:
Add Spanish translation for customisable design padlock page

DeimosHall committed:
Add Spanish translation for the design uart page
- Move images from digital_design/design_uart/images to static/images/design_uart
- Change images path on the english version

DeimosHall committed:
Add Spanish translation for the holidays page

DeimosHall committed:
Add Spanish translation for the logic gates page
- Move images from digital_design/logic_gates/images to static/images/logic_gates
- Change images path on the english version

DeimosHall committed:
Fix tag translation on the design 7 seg, design padlock and wokwi pages

DeimosHall committed:
Add Spanish translation for the wokwi lut generator page
- Move images from digital_design/wokwi_lut_generator/images to static/images/wokwi_lut_generator
- Change images path on the english version

DeimosHall committed:
Add Spanish translation for the logic puzzle page

DeimosHall committed:
Add Spanish translation for the puzzle padlock seq page
- Move images from digital_design/puzzle_padlock_seq/images to static/images/puzzle_padlock_seq
- Change images path on the english version

DeimosHall committed:
Add Spanish translation for the puzzle edge detect page

DeimosHall committed:
Add Spanish translation for the puzzle flipflop page

DeimosHall committed:
Fix tag translation on puzzle pages

wero1414 reviewed (changes requested):
changes request in chat

DeimosHall committed:
Fix translation errors on the following pages
- puzzle adder
- fpga vs asic
- faq
- logic gates
- tt01
- puzzle flipflop
- puzzle edgedetect
- puzzle padlock seq
- design padlock
- design uart
- design 7seg
- wokwi lut generator
- making asics
- important
- testing
- contact
- credits
